### DISCUSSION TITLE
How to visualize all these colors described "flet_core/colors.py" file?

### DISCUSSION THREAD

**Discussion by TheCodeCache** [flet-dev/flet#1022] [Q&A]
`Flet` has consolidated all the supported colors here - [colors.py](https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet-core/src/flet_core/colors.py) file. 

How do we `visualize` all these colors instead of doing trial and check to understand how each of the `color code` looks like 

For example: `lightblue` have below sample shades - 
`lightblue50`
`lightblue100`
`lightblue200`
`lightblue300`
`lightblue400`
`lightblue500` etc.

how do we verify these shades visually? I'm using PyCharm IDE to work with Flet

**Comment by iron3oxide** [comment]
Maybe there should be an overview akin to the icons browser. In the meantime you can refer to the chart at the bottom of [this page](https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors) though.

**Reply by FeodorFitsner** [reply]
https://flutils.pages.dev/ - check "Colors V1" and "Color V2" tabs.
Thanks to @ndonkoHenri

**Comment by TheCodeCache** [comment]
Thanks for the suggestion, that helps
